scripts/e2etests/VSUtils.ps1

```diff
@@ -55,11 +55,20 @@ function GetVSIDEFolderPath
 
 function KillRunningInstancesOfVS
 {
-    $processName = 'devenv'
-    Write-Host "Kill any running instances of $processName..."
-    (Get-Process "$processName" -ErrorAction SilentlyContinue) | Kill -ErrorAction SilentlyContinue
-
-    WaitForProcessExit -ProcessName "$processName" -TimeoutInSeconds 30
+    Get-Process | ForEach-Object {
+        if(-not [string]::IsNullOrEmpty($_.Path)) {
+            $processPath = $_.Path | Out-String
+            if($processPath.StartsWith("C:\Program Files (x86)\Microsoft Visual Studio", [System.StringComparison]::OrdinalIgnoreCase))
+            {
+                Write-Host $processPath
+                Stop-Process $_ -ErrorAction SilentlyContinue -Force
```

Comment:
Based on [docs](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/stop-process?view=powershell-5.1), it seems like this should be - 
`Stop-Process -InputObject $_`

Reply:
it can infer from the variable passed to it.